doc/sources/.static/fresh.css

```diff
@@ -635,6 +635,15 @@ code.docutils{
 	}
 
 }
+.linenos::before {
+	line-height: 0;
+	font-size: 14px;
+	content: attr(data-line-number);
+}
+.linenos {
+	font-size: 0;
+	line-height: 80%;
+}
```

Comment:
Is really needed?

Reply:
wait actually no, as there is no text in linenos now i'll remove it